src/main/resources/config.yml

```diff
@@ -99,15 +102,25 @@ velocity:
   revertedVelocity:
    - SPLASH_POTION
    - ENDER_PEARL
-   - ARROW
 
 
 #This let's you apply a general multiplier to the launch velocity of any kind of projectile. Entry key is the 
 #entity type (https://hub.spigotmc.org/javadocs/spigot/org/bukkit/entity/EntityType.html) and value is the multiplier.
   multiplier:
 #Increase pearl velocity by 20 % for example
+    ENDER_PEARL: 1.0
+    SPLASH_POTION: 1.0
+#These properties only apply to velocity-reverted entities.
```

Comment:
That shouldn't be the case

Reply:
Became the case after I did the ol' copy the NMS method trick, there's now a power parameter, but they seem to do the same thing so one can be removed.